platform/contracts/admin/src/msg.rs

```diff
@@ -67,4 +67,7 @@ pub struct MigrateContracts {
 #[serde(rename_all = "snake_case", deny_unknown_fields)]
 pub enum QueryMsg {
     InstantiateAddress { code_id: CodeId, protocol: String },
+    Protocols {},
+    PlatformAddresses {},
+    ProtocolAddresses { protocol: String },
```

Comment:
1. please document the response type
2. make sure response types are visible outside the crate. They should be consumed by the callers.